transforms/universal/fdedup/spark/src/requirements.txt

```diff
@@ -0,0 +1,8 @@
+pyspark
```

Comment:
Please use >= or <= for the dependencies. Getting the latest is not a good idea.

Reply:
Fixed in commit https://github.com/IBM/data-prep-kit/pull/699/commits/528457c5cc91dad1439c72258be92e8030f45015